pkg/scheduler/apis/config/types.go

```diff
@@ -46,15 +46,8 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta
 
-	// SchedulerName is name of the scheduler, used to select which pods
-	// will be processed by this scheduler, based on pod's "spec.SchedulerName".
-	SchedulerName string
```

Comment:
We need a name for the binary to use for logs/events. something like ```ComponentName``` to avoid confusing it with ```SchedulerName``` used in the pod spec?

Reply:
Events have always being associated to the scheduler name, so I think we should continue to link that to the profile.

As for logs, I don't see the scheduler name being used anywhere.